festim/generic_simulation.py

```diff
@@ -164,6 +165,19 @@ def exports(self, value):
                 "accepted types for exports are list, festim.Export or festim.Exports"
             )
 
+    @property
+    def T(self):
+        return self._T
+
+    @T.setter
+    def T(self, value):
+        if isinstance(value, festim.Temperature):
+            self._T = value
+        elif value is None:
+            self._T = value
+        elif isinstance(value, (int, float, sp.Expr)):
+            self._T = festim.Temperature(value)
```

Comment:
Possibly, an `else` case with an error can be included to add clarity? Similar to https://github.com/festim-dev/FESTIM/blob/fb2c99f593d4f695573c28acf88137ddad252b38/festim/generic_simulation.py#L124-L127

Reply:
added